spec/core/adjust.fmf

```diff
@@ -42,25 +42,37 @@ description: |
     __ https://docs.fedoraproject.org/en-US/ci/test-case-relevancy
 
 example: |
+    # Disable a test for older distros
     enabled: true
     adjust:
         enabled: false
-        when: distro ~< fedora-33
+        when: distro < fedora-33
         because: the feature was added in Fedora 33
 
+    # Adjust the required package name
     require: procps-ng
     adjust:
       - require: procps
         when: distro == centos-6
 
+    # Extend the environment variables, use multiple rules
     adjust:
       - environment+:
             SH: bash
         when: component == bash
         continue: true
-      - when: distro ~< centos-6
+      - when: distro < centos-6
         enabled: false
 
+    # Install the fresh pytest from pip on older distros
```

Comment:
maybe I would like to see there comment with  the default value and reason why there is `order: 90` to show why it is important.

Reply:
I think it would be better to cover these details under the [prepare](https://tmt.readthedocs.io/en/latest/spec/plans.html#prepare) steps specification. Created #697 to improve this.

Reply:
yep fv